query_optimizer/tests/execution_generator/CMakeLists.txt

```diff
@@ -47,4 +47,9 @@ add_test(quickstep_queryoptimizer_tests_executiongenerator_tablegenerator
          "../quickstep_queryoptimizer_tests_ExecutionGeneratorTest"
          "${CMAKE_CURRENT_SOURCE_DIR}/TableGenerator.test"
          "${CMAKE_CURRENT_BINARY_DIR}/TableGenerator.test"
+         "${CMAKE_CURRENT_BINARY_DIR}/")
+add_test(quickstep_queryoptimizer_tests_executiongenerator_stringpatternmatching
```

Comment:
Move this before `quickstep_queryoptimizer_tests_executiongenerator_tablegenerator` according to the alphabet order.